File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/Table.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2019/7/8
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Table {
    private String name;
    private String remarks;

    @Override
    public String toString() {
        return name + "  " + remarks;
    }

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/Table.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2019/7/8
 */

@AllOpen
class Table(var name: String? = null, var remarks: String? = null) {

    constructor() : this("", "")

    override fun toString(): String {
        return "$name  $remarks"
    }
}
